src/cdk/testing/tests/protractor.e2e.spec.ts

```diff
@@ -6,15 +6,17 @@ import {
 } from '@angular/cdk/testing';
 import {expectAsyncError} from '@angular/cdk/testing/private';
 import {ProtractorHarnessEnvironment} from '@angular/cdk/testing/protractor';
-import {browser, by, ElementFinder} from 'protractor';
+import {browser, by, element as protractorElement, ElementFinder} from 'protractor';
 import {MainComponentHarness} from './harnesses/main-component-harness';
 import {SubComponentHarness, SubComponentSpecialHarness} from './harnesses/sub-component-harness';
-import {querySelectorAll as piercingQuerySelectorAll} from 'kagekiri';
 
-const piercingQueryFn =
-    (selector: string, root: ElementFinder) => root.all(by.js(function(this: any) {
-      piercingQuerySelectorAll(selector, this);
-    }));
+// TODO: is there some better way to do this?
```

Comment:
I think it's the right way to do it that way. Kagekiri can never provide types that always reserve the global `kagekiri` namespace. It always depends on whether the UMD bundles have been loaded or not.

I think this boilerplate is reasonable (declaring a const that _needs_ to be named `kagekiri` globally). In an ideal world, it would just refer to an official type of `kagekiri` for type safety.

It would be also possible to avoid the boilerplate by just passing in a string to `by.js`. That is less type-safe, but given that we don't have official kagekiri types anyway, it seems to be unsafe anyway. I still prefer the current way though.